src/Microsoft.Azure.WebJobs.Host/Bindings/Path/BindingTemplate.cs

```diff
@@ -84,54 +87,54 @@ public static BindingTemplate FromString(string input, bool ignoreCase = false)
         /// <returns>Resolved string if succeeded.</returns>
         /// <exception cref="InvalidOperationException">Thrown when required parameter value is not available.
         /// </exception>
-        public string Bind(IReadOnlyDictionary<string, string> parameters)
+        public string Bind(IReadOnlyDictionary<string, object> parameters)
```

Comment:
given the extension method is in a different namespace; I think an adding an instance method is actually a good way to go moving forward. 

---
In reply to: [112510756](https://github.com/Azure/azure-webjobs-sdk/pull/1117#discussion_r112510756) [](ancestors = 112510756,112509482,112491637)